src/__tests__/helper.ts

```diff
@@ -143,3 +144,9 @@ export const mineBlockAndSync = async ({ lnds, blockHeight }: { lnds: Array<any>
   }
   await Promise.all(promiseArray)
 }
+
+export const set2FA = async ({ wallet, secret }) => {
+  const token = generateToken(secret)!.token
```

Comment:
is there a way to type this in a way we don't need the `!`?

Reply:
I tried to not have the `!` but it seems because of the way `generateToken` is implemented in the `node-2fa` library, the value returned can indeed be null.